dkg/pedersen/mod_test.go

```diff
@@ -0,0 +1,128 @@
+package pedersen
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.dedis.ch/fabric/dkg"
+	"go.dedis.ch/fabric/mino"
+	"go.dedis.ch/fabric/mino/minoch"
+	"go.dedis.ch/fabric/mino/minogrpc"
+	"go.dedis.ch/fabric/mino/minogrpc/routing"
+	"go.dedis.ch/kyber/v3"
+)
+
+func TestPedersen_Scenario(t *testing.T) {
+	n := 10
+
+	addrFactory := minoch.AddressFactory{}
```

Comment:
Why are you using this ? It should work with the `fake.Address` like skipchain and byzcoin.

Reply:
Actually, the address should come from the instance of Mino. When you instantiate an implementation of Mino, it creates its own address and you should get it from there.

Reply:
```
m := minogrpc.NewMinogrpc("0.0.0.0", 2000, treeFactory) // it should be done like that
addrs[i] = m.GetAddress()
```